package.json

```diff
@@ -7,15 +7,17 @@
         "app/*",
         "client/**",
         "core/*",
-        "server/*"
+        "server/*",
+        "server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming"
     ],
     "scripts": {
         "prepare": "husky install .husky",
         "lint": "eslint app/ client/ core/ script/ server/ --ext .ts,.tsx && prettier --check .",
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
         "clean": "ts-node ./script/clean.ts",
-        "compile": "tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "compile": "npm run steamingClient &&  tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "steamingClient": "npm run build -w server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming",
```

Comment:
Can we do `compile-streamingClient` (or something similar) here to expose the action?